builder/googlecompute/step_start_tunnel.go

```diff
@@ -51,14 +51,17 @@ type IAPConfig struct {
 	// What file extension to use for script that sets up gcloud.
 	// Default: ".sh"
 	IAPExt string `mapstructure:"iap_ext" required:"false"`
+	// How long to wait, in seconds, before assuming a tunnel launch was
+	// successful. Defaults to 30 seconds.
+	IAPTunnelLaunchTimeout int `mapstructure:"iap_tunnel_launch_timeout" required:"false"`
```

Comment:
Would the features of `time.Duration` bring value here ? Allowing users to set this value with a string like: `30s`/`1m` etc.

Reply:
The timeout is the number of times we run through a loop with a 1s wait. It isn't used as a time.Duration in a client timeout call.

Reply:
Oh, right, yes, I read that doc before reading the code ! I think the description in the previous comment you made is great !

Reply:
I thought about that, and while it isn't technically a timeout, having a max 1s retries is essentially the same thing but stated in a less intuitive way.

Reply:
though it isn't actually a timeout, it's a launch wait... maybe "tunnel_launch_wait"

Reply:
naming is hard 😬  !  I like `tunnel_launch_wait` !

Reply:
Hahah sounds good. I'll change it to tunnel_launch_wait which is probably the clearest.